core/server/master/src/main/java/alluxio/master/metastore/rocks/RocksStore.java

```diff
@@ -84,10 +84,11 @@ public final class RocksStore implements Closeable {
    * @param dbOpts the configured RocksDB options
    * @param columnFamilyDescriptors columns to create within the rocks database
    * @param columnHandles column handle references to populate
+   * @param reset if true, the RocksDB will be cleaned up and then new DB will be created
    */
   public RocksStore(String name, String dbPath, String checkpointPath, DBOptions dbOpts,
       Collection<ColumnFamilyDescriptor> columnFamilyDescriptors,
-      List<AtomicReference<ColumnFamilyHandle>> columnHandles) {
+      List<AtomicReference<ColumnFamilyHandle>> columnHandles, boolean reset) {
```

Comment:
In the original implementation, the RocksDB always resets (removed if exists, created a new DB),
because Master is going to replay all the records from journal.
But in Dora Worker, we need all the metadata to survive restart.